hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -75,55 +111,27 @@ public EvictionConfig() {
     }
 
     public EvictionConfig(int size, MaxSizePolicy maxSizePolicy, EvictionPolicy evictionPolicy) {
-        /**
```

Comment:
It's not common proxies to use setters in a constructor. So there is actually no need for a dangeling JavaDoc comment (it doesn't belong in a method body), which creates an IDEA warning. Also no other config classes beside the copy&paste `NearCachePreloaderConfig` has those comments.